### PR TITLE
WMS-728 | Docs in CI

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?
* Remove no longer required fix in CI
* Add Docs to CI
* Fix all docs realted problems...

#### :man_shrugging: Why?
* Because PRs are being pushed breaking docs.

#### :pushpin: Jira Issue
[WMS-728](trust-me)

#### :no_good: Blocked by
Not blocked.

#### :clipboard: Pending
- [x] Fix all docs.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,8 +39,11 @@ jobs:
      - name: Run linter
        run: npx elm-format --validate src/ showcase/
 
+     - name: Run docs
+       run: npx elm-format --validate src/ showcase/
```
Comment (miguelcoba) [inline .github/workflows/ci.yml:43]: this does the same as the linter, is that correct?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -39,8 +39,11 @@ jobs:
...
+
      - name: Run analyse
-       run: npx elm-analyse
```
Comment (miguelcoba) [inline .github/workflows/ci.yml:43]: why are you removing the analyse?

Comment (miguelcoba) [reply .github/workflows/ci.yml:43]: maybe the github diff is off but I see it this way:
image

Comment (PedroHLC) [reply .github/workflows/ci.yml:43]: My mistake, I edited the wrong line.

**Review by miguelcoba — APPROVED** [review]

**Review by renanpvaz — APPROVED** [review]

**Pull request merged**